books/RayTracingInOneWeekend.html

```diff
@@ -596,29 +604,28 @@
 Ray-Sphere Intersection
 ------------------------
 <div class='together'>
-Recall that the equation for a sphere centered at the origin of radius $R$ is $x^2 + y^2 + z^2 =
-R^2$. Put another way, if a given point $(x,y,z)$ is on the sphere, then $x^2 + y^2 + z^2 = R^2$. If
-the given point $(x,y,z)$ is _inside_ the sphere, then $x^2 + y^2 + z^2 < R^2$, and if a given point
-$(x,y,z)$ is _outside_ the sphere, then $x^2 + y^2 + z^2 > R^2$.
+Recall that the equation for a sphere with a radius of $r$ and that is centered at the origin is
+$x^2 + y^2 + z^2 = r^2$. You can also think of this as saying that if a given point $(x,y,z)$ is on
+the sphere, then $x^2 + y^2 + z^2 = r^2$. If a given point $(x,y,z)$ is _inside_ the sphere, then
+$x^2 + y^2 + z^2 < r^2$, and if a given point $(x,y,z)$ is _outside_ the sphere, then
+$x^2 + y^2 + z^2 > r^2$.
 
-It gets uglier if the sphere center is at $(C_x, C_y, C_z)$:
+It gets uglier if the sphere center isn't at the origin, _i.e._ $(c_x, c_y, c_z)$:
 
-  $$ (x - C_x)^2 + (y - C_y)^2 + (z - C_z)^2 = r^2 $$
+  $$ (x - c_x)^2 + (y - c_y)^2 + (z - c_z)^2 = r^2 $$
 </div>
 
 <div class='together'>
-In graphics, you almost always want your formulas to be in terms of vectors so all the x/y/z stuff
-is under the hood in the `vec3` class. You might note that the vector from center
-$\mathbf{C} = (C_x,C_y,C_z)$ to point $\mathbf{P} = (x,y,z)$ is $(\mathbf{P} - \mathbf{C})$, and
-therefore
+In graphics, you almost always want your formulas to be in terms of vectors so that all the
+$x$/$y$/$z$ stuff can be simply represented using a `vec3` class. You might note that the vector
+from center $\mathbf{C} = (c_x, c_y, c_z)$ to point $\mathbf{P} = (x,y,z)$ is
```

Comment:
I think we should continue with Cx, Cy and Cz, the components of *C* — that is, points capitalized and bold, components normal weight but subscripted from the capitalized letter.

Reply:
ugh. Wow I hate it.
I understand why AABB has it set up the way that it is, but I can't help but look at that and see a problem.

Basically, we don't have a set syntax for the names of vectors.

I don't disagree with your comment, even though I would vastly prefer lower case here, so I'll be making the change as per your recommendation